aperture/request_handler.cpp

```diff
@@ -164,10 +164,12 @@ namespace http {
 			// grab the query string parts
 			QueryString queryString = decode_query_string(req.uri);
 
-			if (queryString.find("texture_id") == queryString.end() &&
-				queryString.find("mesh_id") == queryString.end())
+			bool isTextureRequest = queryString.find("texture_id") != queryString.end();
+			bool isMeshRequest = queryString.find("mesh_id") != queryString.end();
+
```

Comment:
Everything looks great. I just wanted to point out that this splitting the complex conditional into separate assignments is a semantic change. Previously it didn't evaluate the `isMeshRequest` expression unless is the `isTextureRequest` expression was false. Slight performance penalty, but more than made up for by the re-use of the booleans below.  Also much more readable and maintainable so I would have done exactly the same thing.  Merge away, looks good to go!